wp-content/themes/goonj-crm/assets/styles/main.scss

```diff
@@ -470,6 +470,203 @@ form input {
   background-color: #ccc;
 }
 
+// CSS for the Contribution Page
+#crm-main-content-wrapper {
```

Comment:
Can we move all these styles into a new file called `contribution.scss`

and then do `@import 'contribution';`